github-workflow-roles/variables.tf

```diff
@@ -25,7 +25,7 @@ variable "repository_name" {
 variable "role_path" {
   description = "Path of IAM role"
   type        = string
-  default     = "/"
+  default     = "/RVM/"
```

Comment:
Consider adding a warning not to change this, since the `assumed_role` resource assumes this path (`/RVM/`) will be used.

Reply:
I think we should hardcode this into the code as opposed to having it as a variable